src/components/Global/AddressLink/index.tsx

```diff
@@ -0,0 +1,28 @@
+import Link from 'next/link'
+import { useEffect, useState } from 'react'
+
+import * as utils from '@/utils'
+
+const AddressLink = ({ address }: { address: string }) => {
```

Comment:
Thought: this is a great utility. What's your thoughts on using this throughout the app?